src/cpp/src/text2image/models/autoencoder_kl.cpp

```diff
@@ -43,7 +44,7 @@ AutoencoderKL::AutoencoderKL(const std::string& root_dir,
                 const std::string& device,
                 const ov::AnyMap& properties)
     : AutoencoderKL(root_dir) {
-    compile(device, properties);
+    compile(device, filter_out_adapters_from_properties(properties).value_or(properties));
```

Comment:
looks like `.value_or(properties)` should be hidden inside `filter_out_adapters_from_properties`